encrypted-media/clearkey-mp4-playback-temporary-multikey-sequential-readyState.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
```

Comment:
I think the short timeout is 10 seconds. For these tests we expect to see 5 seconds of playback, in order to traverse the portions of the media with different keys. Since there is also a server round-trip, initialization etc. 10 seconds might be a bit close.